### PR TITLE
RE-1276 Add rpc-maas projects using standard jobs

### PR DESCRIPTION
**Pull request by git-harry**
These new jobs are designed to replace the IRR jobs for rpc-maas. Once
these jobs have been added the equivalent IRR jobs will be removed in a
separate commit.

Issue: [RE-1276](https://rpc-openstack.atlassian.net/browse/RE-1276)

### CODE REVIEW

File: rpc_jobs/rpc_maas.yml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    series: "master"
+    branches:
+      - "master.*"
+    image:
+      - xenial:
+          IMAGE: "Ubuntu 16.04 LTS (Xenial Xerus) (PVHVM)"
+    scenario:
+      - master
+      - pike
+      - ocata
```
Comment (odyssey4me) [inline rpc_jobs/rpc_maas.yml]: In an effort to save some resources, I suggest adding ``TRIGGER_PR_PHRASE_ONLY: true`` to ocata, pike & master as they are not required PR's in the repo.

File: rpc_jobs/rpc_maas.yml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+      - newton
+      - ceph
+    action:
+      - deploy:
+          FLAVOR: "performance2-15"
```
Comment (odyssey4me) [inline rpc_jobs/rpc_maas.yml]: The original jobs use "general1-8" for all scenario's except for master which uses "performance2-15". Can this be adjusted to do the same in order to save some resourcing.

Comment (git-harry) [reply rpc_jobs/rpc_maas.yml]: Done.

Good spot, I don't remember doing this intentionally.

File: rpc_jobs/rpc_maas.yml (hunk 1 above)
Comment (git-harry) [reply rpc_jobs/rpc_maas.yml]: I've done this for master, ocata and kilo. I think pike should be required given the current state of rpc-openstack so I'll make that so.

**Comment by git-harry** [comment]
@mattt416 I've added a couple of requests for reviews.

@cloudnull / @tonytan4ever this change is to move from using IRR to the standard jobs, mostly it is a like for like swap however given that a few of the jobs are not required by rpc-maas I've made them require a manual trigger. Please will one of you give this a review so that someone working on that repository has verified the change.

**Pull request merged**